readthedocs/api/v2/templates/restapi/footer.html

```diff
@@ -1,11 +1,12 @@
 <!-- Inserted RTD Footer -->
+{% load i18n %}
 <div class="injected">
 
   {% if not new_theme %}
   <div class="rst-versions rst-badge" data-toggle="rst-versions">
     <span class="rst-current-version" data-toggle="rst-current-version">
       <span class="fa fa-book">&nbsp;</span>
-      v: {{ current_version }}
+         {{ current_version }}
```

Comment:
This shouldn't be changed

Reply:
You are seeing old edit - https://i.gyazo.com/bca5cf1cbf6cff25da2603cbed0a5b4a.png.

Reply:
The diff is showing that `v:` was removed, that shouldn't be removed.

Reply:
I ignored that, fixed.